public/backend/js/dataTable.js

```diff
@@ -0,0 +1,4 @@
+$(document).ready(function(){
+  var frmId = $('table').attr('id');
```

Comment:
[rank C] why do you update this code like that? What's happen if we have many table elements in our page? If you cannot understand my opinions, please tell me to clarify.